packages/ra-core/src/core/CoreAdminRoutes.spec.tsx

```diff
@@ -247,61 +250,27 @@ describe('<CoreAdminRoutes>', () => {
                 getPermissions: jest.fn().mockResolvedValue(''),
             };
 
-            const history = createMemoryHistory();
             render(
-                <CoreAdminContext
-                    authProvider={authProvider}
-                    dataProvider={testDataProvider()}
-                    history={history}
-                >
-                    <CoreAdminRoutes
-                        layout={Layout}
-                        loading={Loading}
-                        catchAll={CatchAll}
-                        requireAuth
+                <MemoryRouter>
+                    <CoreAdminContext
+                        authProvider={authProvider}
+                        dataProvider={testDataProvider()}
                     >
-                        <Resource name="posts" list={() => <i>PostList</i>} />
-                    </CoreAdminRoutes>
-                </CoreAdminContext>
+                        <CoreAdminRoutes
+                            layout={Layout}
+                            loading={Loading}
+                            catchAll={CatchAll}
+                            requireAuth
+                        >
+                            <Resource
+                                name="posts"
+                                list={() => <i>PostList</i>}
+                            />
+                        </CoreAdminRoutes>
+                    </CoreAdminContext>
+                </MemoryRouter>
             );
             expect(screen.queryByText('PostList')).toBeNull();
-            expect(screen.queryByText('Loading')).toBeNull();
-            resolve();
-            await waitFor(() =>
-                expect(screen.queryByText('PostList')).not.toBeNull()
-            );
-        });
-        it('should show a loader when requireAuth is true and dataProvider.checkAuth() takes more than 1s to reply', async () => {
```

Comment:
This was testing that CoreAdminRouter displays the loader only after 1s. Now, this feature has moved to the Loader itself.